Deprecated Packs Clarification (#2682)
spectrocloud/librarium: docs/docs-content/integrations/maintenance-policy.md

  the pack. Active and deployed clusters are not affected as the pack is still available in Palette.

  The pack remains in _Deprecated_ state for three months before it moves to _Disabled_ state.

- **Disabled**: When a pack is disabled, it is no longer available for selection in Palette. When creating new profiles,
  you must use a newer version of the pack. You can still launch new clusters using existing profiles that contain the
  disabled pack. Active and deployed clusters are not affected as the pack is still available in Palette.

  The pack remains in _Disabled_ state for three months before it is deleted.

- **Deleted**: When a pack is deleted, it is removed from Palette. You will not be able to deploy a new cluster with a
  cluster profile containing the deleted pack or create a new cluster profile referencing the deleted pack. We recommend
  that you update your existing cluster profiles to use a newer version of the pack and apply the changes to your active
  clusters. Active clusters using the deleted pack will typically not experience issues if a node is rebooted or if a

>   new node is added to the cluster. However, if problems exist with the deleted pack version, such as upstream bugs. In

Reviewer: ```suggestion
  new node is added to the cluster. However, if there are any problems with the deleted pack version, such as upstream bugs,
```

>   that case, you may encounter issues if they affect scaling operations or other cluster operations.

Reviewer: ```suggestion
  you may encounter issues if they affect scaling or other cluster operations.
```
